bin/build-locales

```diff
@@ -39,5 +41,35 @@ poFiles.forEach((pofile) => {
     format: 'jed1.x',
     fuzzy: config.get('po2jsonFuzzyOutput'),
   });
-  fs.writeFileSync(jsonfile, json);
+  const localeObject = JSON.parse(json);
+
+  // Add the moment locale JS into our locale file, if one is available and
+  // we're building for AMO (which is the only app that uses moment right
+  // now).
+  if (appName === 'amo') {
+    var defineLocale = null;
+    try {
+      const momentLocale = locale.replace('_', '-').toLowerCase();
+      // We're using `new Function()` here to create a function out of the
+      // raw code in the file; if we require it here it will evaluate here
```

Comment:
I think mentioning why this is being wrapped in a function more explicitly might be nice. Something like:

```js
// We're using `new Function()` here to create a function out of the
// raw code in the file; this function won't be executed but will be
// written out by `toSource()` so that it can be used later.